editor.planx.uk/src/pages/FlowEditor/components/Settings/GeneralSettings/BoundaryForm.tsx

```diff
@@ -21,12 +29,23 @@ export default function BoundaryForm({ formikConfig, onSuccess }: FormProps) {
     ...formikConfig,
     validationSchema: formSchema,
     onSubmit: async (values, { resetForm }) => {
-      const isSuccess = await useStore.getState().updateTeamSettings({
-        boundaryUrl: values.boundaryUrl,
-      });
-      if (isSuccess) {
-        onSuccess();
-        resetForm({ values });
+      try {
+        const { data } = await axios.get(`${values.boundaryUrl}.geojson`);
+        const bboxPoly = bboxPolygon(bbox(data));
+        const bboxFeature = feature(bboxPoly.geometry);
```

Comment:
I'm not quite sure if this is the simplest way to achieve this. If we add types and maybe wrap this in a descriptive helper function it may help clear this up.

Reply:
Addressed here - [74e7089](https://github.com/theopensystemslab/planx-new/pull/3378/commits/74e708982d93980efd3fc5141d7f669a74bc90d0)

The confusion may have come from the fact that there's a `bbox` property on the object, which sort of looks wrong if you're not expecting it. This is actually fine and is an optional part of the GeoJSON spec ([docs](https://datatracker.ietf.org/doc/html/rfc7946#autoid-23)).

![image](https://github.com/theopensystemslab/planx-new/assets/20502206/57aedf45-77da-4bed-9339-918e10f60e38)